ses/variables.tf

```diff
@@ -18,4 +18,9 @@ variable "email_address" {
 
 variable "hosted_zone_id" {
   description = "Hosted zone ID"
-}
\ No newline at end of file
+}
+
```

Comment:
This is the same as I said on the other pull request but I'm not sure what this variable and the `manual_creation` one is doing.

Reply:
thanks, corrected